src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -480,6 +466,10 @@ export class DotEditContentHtmlService {
         return contentletEventsMap[event];
     }
 
+    private shouldUpdateContentletInode(target: HTMLElement) {
+        return target.dataset.dotObject === 'edit-content' && target.tagName === 'BUTTON';
+    }
+
```

Comment:
why do we need this check?

Reply:
Becuase we only need to update the contentlet inode in the DOM when we're editing from the pencil edit button at the top, not if we edit the vtl or the shortcut edit in a listing like in news.

Reply:
but  the value of dotObject is 'contentlet' in that case, isn't it?

Reply:
`target` will be always a button to edit.